src/window.rs

```diff
@@ -271,6 +317,13 @@ impl HieroglyphicWindow {
                         classifications
                             .into_iter()
                             .take(25)
+                            .filter(|s| {
+                                // only display symbols supported by the current language
+                                classify::Symbol::from_id(s)
+                                    .unwrap()
+                                    .command(&language_mode)
+                                    .is_some()
```

Comment:
I'm not super happy about this additional roundtrip to `Symbol` (i.e. `str` → `Symbol` → `StringObject` → `Symbol`), but I would need to think about how to do it better.